tests/queries/test_explain.py

```diff
@@ -90,7 +92,7 @@ def test_mysql_analyze(self):
         with CaptureQueriesContext(connection) as captured_queries:
             qs.explain(analyze=True, format='JSON')
         self.assertEqual(len(captured_queries), 1)
-        if connection.mysql_is_mariadb:
+        if connection.mysql_is_mariadb or self.connection.mysql_version >= (8, 0, 21):
```

Comment:
This test fails: `(1235, "This version of MySQL doesn't yet support 'FORMAT=JSON with EXPLAIN ANALYZE'")`.

Reply:
Yes, of course, sorry.